org.eclipse.xtext.web.example.entities/src-gen/org/eclipse/xtext/web/example/entities/services/EntitiesGrammarAccess.java

```diff
@@ -770,8 +776,8 @@ public ParserRule getXListLiteralRule() {
 	//XClosure XExpression:
 	//	=> ({XClosure}
 	//	'[')
-	//	=> ((declaredFormalParameters+=JvmFormalParameter (',' declaredFormalParameters+=JvmFormalParameter)*)?
-	//	explicitSyntax?='|')?
+	//	=> ((declaredFormalParameters+=JvmFormalParameter (',' declaredFormalParameters+=JvmFormalParameter)*)
+	//	? explicitSyntax?='|')?
```

Comment:
Hmm this is really a bad mis-formatting. There should never be a space before the cardinality according to https://github.com/eclipse/xtext-core/blob/master/org.eclipse.xtext/src/org/eclipse/xtext/xtext/XtextFormatterJava.java#L204

Reply:
maybe we miss some traverals

Reply:
@ArneDeutsch this one still looks bad

Reply:
Will be fixed with next commit. Allow autowrap after question mark as well.